ui/templates/base.html

```diff
@@ -7,6 +7,10 @@
     <script type="text/javascript">
     var SETTINGS = {{ js_settings_json|safe }};
     </script>
+    {% if style_public_src %}
+    <script type="text/javascript" src="{{ style_public_src }}"></script>
+    <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
```

Comment:
I'm a bit worried about race conditions if it's not in `public.js`. The CDN might take a little longer than our JS blob to download.